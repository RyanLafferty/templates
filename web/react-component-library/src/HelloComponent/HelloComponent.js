import React, { Component } from 'react';


class HelloComponent extends Component {
  render() {
    return (
      <div>
        Hello World!
      </div>
    );
  }
}

export default HelloComponent;