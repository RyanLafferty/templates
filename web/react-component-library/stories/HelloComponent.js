import React from 'react';
import { storiesOf } from '@storybook/react';
import HelloComponent from './../src/HelloComponent';


storiesOf('HelloComponent', module)
.add('HelloComponent', () => (
  <HelloComponent/>
));
