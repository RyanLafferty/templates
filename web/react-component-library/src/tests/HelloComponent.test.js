import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import HelloComponent from 'HelloComponent';

describe('HelloComponent', () => {
  let props;
  const renderer = new ShallowRenderer();

  beforeEach(() => {
    props = {};
  });

  describe('when no props are provided', () => {
    it('HelloComponent renders', () => {
      expect(
        renderer.render(<HelloComponent {...props} />),
      ).toMatchSnapshot();
    });
  });
});
