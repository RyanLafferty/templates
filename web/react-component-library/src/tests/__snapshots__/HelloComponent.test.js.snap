// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloComponent when no props are provided HelloComponent renders 1`] = `
<div>
  Hello World!
</div>
`;
